semantic_conventions/metrics/process-metrics.yaml

```
@@ -0,0 +1,138 @@
+groups:
+  - id: process.cpu
```

```suggestion
  - id: attributes.process.cpu
```

I started adding `attributes.` to `attribute_group` to easily spot that this is not a span/metric/log convention but just rather a "bag" of attributes to be shared.